example_problems/hello/submissions/accepted/extra-file-awk/test-hello.awk

```diff
@@ -1,4 +1,4 @@
-# This should give RUN-ERROR on the default problem 'hello', since it
+# This should give CORRECT on the default problem 'hello',
```

Comment:
super-nit: in the commit message of this file, you are fixing the comment, not the annotation.